src/applications/mhv-secure-messaging/containers/Compose.jsx

```diff
@@ -89,6 +104,41 @@ const Compose = () => {
     },
     [header, acknowledged, pageTitle],
   );
+  // make sure the thread list is fetched when navigating to the compose page
+  useEffect(
+    () => {
+      if (!threadList || threadList.length === 0) {
```

Comment:
the most common flow is to navigate to compose page from the inbox, where threadsList will already be fetched for Inbox folder with 10 threads per page. This does not satisfy criteria below and per condition above - `getListOfThreads` will not be triggered

Reply:
Great callout! 

We only want to load the sent folder on page load (or not if the sent folder is already loaded) 

Is there an already solved way to check what folder is loaded in redux?

Reply:
`state => state.sm.folders.folder` is where the current folder is dispatched to.